src/sipi.cpp

```diff
@@ -862,6 +862,7 @@ int main(int argc, char *argv[])
       }
     } catch (Sipi::SipiImageError &err) {
       std::cerr << err << std::endl;
+      exit(1);
```

Comment:
Hmmm, does that mean that in the past, all `SipiImageError` resulted in return code 0, but now, they will all result in return code 1? 

That would not be the expected behaviour. Because often, an image is only slightly corrupted, but perfectly readable by most viewers. For example, the ca. 66'000 images of LHTT produced ca. 14'000 error logs. But most of them are negligible, because there is no real defect in the image. We'd like that SIPI only aborts with non-0 in these very specific cases:

- `Corrupt JPEG data: premature end of data segment` (https://linear.app/dasch/issue/DEV-4480)
- `Corrupt JPEG data: bad Huffman code` (https://linear.app/dasch/issue/DEV-4457)

Reply:
Very odd, but it was also my intention to check whether this new behavior breaks the build. Unfortunately the `main` build is broken. 🤦

Anyhow, I'll restrict it to the behavior you've described.